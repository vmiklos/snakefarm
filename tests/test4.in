debug 1
out 1
time 100
load test4.map
addplayer 1
addsnake 1 1 0 1 1 0
7 0
endsnake
addplayer 2
addsnake 2 2 0 1 1 0
9 0
endsnake
step
comment # itt elvileg veget ert szabalyosan az elso palya, betoltjuk a masodikat (a testcase 1 palyajat)
load test1.map
comment # itt az elozo palyan mar hasznalt playerid es snakeid kerul kiosztasra, ennek nem szabad hibat okoznia
addplayer 1
addsnake 1 1 0 1 1 0
6 0
endsnake
exit