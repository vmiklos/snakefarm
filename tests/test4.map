1;W;0;0
2;W;0;1
3;W;0;2
4;W;0;3
5;W;0;4

6;W;1;0
7;0;2;6
8;W;3;7
9;0;4;8
10;W;5;9

11;W;6;0
12;W;7;11
13;W;8;12
14;W;9;13
15;W;10;14

