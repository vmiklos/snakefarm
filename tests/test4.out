Event MapLoad test4.map
Event Step 1
StepEvent Snake_Wall
StepEvent SnakeDie
End Step
Event Step 2
StepEvent Snake_Wall
StepEvent SnakeDie
End Step
# itt elvileg veget ert szabalyosan az elso palya, betoltjuk a masodikat (a testcase 1 palyajat)
Event MapLoad test1.map
# itt az elozo palyan mar hasznalt playerid es snakeid kerul kiosztasra, ennek nem szabad hibat okoznia
Event Exit
